rotkehlchen/rotkehlchen.py

```diff
@@ -1028,6 +1028,7 @@ def query_periodic_data(self) -> dict[str, Union[bool, list[str], Timestamp]]:
             with self.data.db.conn.read_ctx() as cursor:
                 result['last_balance_save'] = self.data.db.get_last_balance_save_time(cursor)
                 result['connected_eth_nodes'] = [node.name for node in self.chains_aggregator.ethereum.node_inquirer.get_connected_nodes()]  # noqa: E501
+                result['connected_optimism_nodes'] = [node.name for node in self.chains_aggregator.optimism.node_inquirer.get_connected_nodes()]  # noqa: E501
```

Comment:
shouldn't this also have an api docs change? No other tests use this apart from the one you have the empty list in?